main.rb

```diff
@@ -0,0 +1,8 @@
+require_relative 'app'
+
+def main
+  app = App.new
+  app.run
+end
```

Comment:
- [ ] Dear @sergio-usma, Your app works fine till the point where I add a person and a book, I also confirm the rental is working fine. However, once I added those details, I could not see a list of books, the app kept crashing when I tried to list available books. Please make sure it's possible to see a list of books when a user selects option 1
![Screen Shot 2023-09-10 at 12 30 00 AM](https://github.com/sergio-usma/ruby_school_library/assets/69077061/9ecbc60a-d13b-4b14-9c45-ed8b6df2110e)

Reply:
Ok bro! The bug was solved!!! Thank you very much for your review!!